### PR TITLE
feat: Add ability to run promtail for log collection

### PR DESCRIPTION
**Pull request by nahsi**

### CODE REVIEW

File: molecule/converge.yml [hunk 1]
```diff
@@ -3,8 +3,12 @@
   become: true
   vars:
     nox_version: "0.22.0"
-    ipfs_cli_version: "0.25.0"
     ccp_version: "0.2.0"
+    ipfs_cli_version: "0.25.0"
```
Comment (folex) [inline molecule/converge.yml:7]: 0.26.0?

File: roles/promtail/README.md
Comment (folex) [inline roles/promtail/README.md:1]: 🔥

Comment (folex) [inline roles/promtail/README.md:1]: Would be great to mention how to check if it works. Like, what to see in promtail's logs, maybe where to go to Grafana.

File: molecule/converge.yml (hunk 1 above)
Comment (nahsi) [reply molecule/converge.yml:7]: Nah doesn't matter. It is just for tests.

File: roles/promtail/README.md
Comment (nahsi) [reply roles/promtail/README.md:1]: I think that it should be somewhere else in the doc as this role doesn't setup grafana from providers.

**Pull request merged**